nixos/modules/virtualisation/libvirtd.nix

```
@@ -426,9 +426,11 @@ in
             libvirt/qemu/networks/*.xml \
             libvirt/nwfilter/*.xml );
         do
-            mkdir -p /var/lib/$(dirname $i) -m 755
-            if [ ! -e /var/lib/$i ]; then
-              cp -pd ${cfg.package}/var/lib/$i /var/lib/$i
+            # Intended behavior
+            # shellcheck disable=SC2174
+            mkdir -p "/var/lib/$(dirname "$i")" -m 755
```

```suggestion
            mkdir "/var/lib/$(dirname "$i")" -m 755 || true
```
we not really even want to create /var/lib/, right?

---

Yes, but the for loop includes files that are two folders deep. I thought about rewriting this code using `install` perhaps.

---

Let's do this another time